core/chains/evm/gas/block_history_estimator.go

```diff
@@ -307,17 +307,19 @@ func (b *BlockHistoryEstimator) Recalculate(head *evmtypes.Head) {
 
 	percentile := int(b.config.BlockHistoryEstimatorTransactionPercentile())
 
+	lggr := b.logger.With("head", head)
+
 	if len(b.rollingBlockHistory) == 0 {
-		b.logger.Debug("No blocks in history, cannot set gas price")
+		lggr.Debug("No blocks in history, cannot set gas price")
```

Comment:
I am wondering why sometimes we start a sentence from letter case and sometimes with capital case. Since this is the first PR of this kind, shall not we unify this across all the code?

Reply:
Good point, probably it should always start with a capital.